crates/cheatcodes/src/utils.rs

```diff
@@ -149,3 +149,25 @@ impl Cheatcode for resumeTracingCall {
         Ok(Default::default())
     }
 }
+impl Cheatcode for setArbitraryStorageCall {
+    fn apply_stateful<DB: DatabaseExt>(&self, ccx: &mut CheatsCtxt<DB>) -> Result {
+        let Self { target } = self;
+        ccx.state.arbitrary_storage.push(*target);
+
+        Ok(Default::default())
+    }
+}
+
+impl Cheatcode for copyStorageCall {
+    fn apply_stateful<DB: DatabaseExt>(&self, ccx: &mut CheatsCtxt<DB>) -> Result {
+        let Self { from, to } = self;
+        if let Ok(from_account) = ccx.load_account(*from) {
```

Comment:
this won't work for forking but Ig it's out of scope?

Reply:
@klkvr please check https://github.com/foundry-rs/foundry/pull/8807/commits/16f068f6e9f557571464b9ad5d39d9f24ff40bb5 Had to accommodate copy from arbitrary storage scenario pointed here https://github.com/foundry-rs/foundry/pull/8807#issuecomment-2339014187 too, so a little bit bigger change, ptal. thanks!